services/QuillLMS/client/app/bundles/Staff/components/evidence/EvidenceLanding.tsx

```diff
@@ -14,6 +15,7 @@ const EvidenceLanding = () => (
       <Route component={UniversalRule} path='/universal-rules/:ruleId' />
       <Route component={UniversalRulesIndex} path='/universal-rules' />
       <Route component={Activities} path='/activities' />
+      <Route component={Hints} path='/hints/:hintId?' />
```

Comment:
Should that question mark be at the end there?

Reply:
So according to this StackOverflow article (https://stackoverflow.com/questions/35604617/react-router-with-optional-path-parameter), React Router doesn't document this particularly well, but the `?` after a param in a route means that the param is optional.  Doing it this way means that `#/hints` and `#/hints/123` both load the same component `Hints`.  There might be another way to do this that's more in line with our existing approaches, but I didn't see anything obvious in my survey of routes.